static/app/views/projectsDashboard/projectChart.tsx

```diff
@@ -185,15 +182,13 @@ function ProjectChart({firstEvent, stats, transactionStats, onBarClick, project}
   );
 }
 
-const constructErrorsLink = (
-  organization: Organization,
-  project: Project,
-  timestamp: number
-) => {
+const constructErrorsLink = (project: Project, timestamp: number) => {
   const start = new Date(timestamp);
   const end = new Date(start);
   end.setHours(end.getHours() + 1);
-  return `/organizations/${organization.slug}/issues/?project=${project.id}&start=${start.toISOString()}&end=${end.toISOString()}`;
+  return normalizeUrl(
+    `/issues/?project=${project.id}&start=${start.toISOString()}&end=${end.toISOString()}`
```

Comment:
Oh i think you actually still need to pass the org when you use normalize url lol